### PR TITLE
Change RadioTV Oriente link

### PR DESCRIPTION
**Pull request by sguinetti**
This PR update peruvian channel. The old link is a unknow sport channel without other information.

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -184,7 +184,7 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtu
 #EXTINF:-1 tvg-id="RadioXtrema.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="" group-title="Music",Radio Xtrema [Not 24/7]
 https://rtmp02.portalexpress.es/xtrematv2/xtrematv2/playlist.m3u8
 #EXTINF:-1 tvg-id="RadioTVOrienteYurimaguas.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/www.roriente.org/picture?width=320&height=320" group-title="Local",RadioTV Oriente (Yurimaguas) (360p) [Not 24/7]
```
Comment (freearhey) [inline channels/pe.m3u:186]:
```suggestion
#EXTINF:-1 tvg-id="RadioTVOrienteYurimaguas.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/www.roriente.org/picture?width=320&height=320" group-title="Local",RadioTV Oriente (Yurimaguas) (1080p)
```

**Review by freearhey — CHANGES_REQUESTED** [review]
It would be better to leave both links in the playlist, just changing the name of the old one.

**Review by Dum4G — APPROVED** [review]

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u]: I forgot it. Thank you for this suggestion.

**Comment by sguinetti** [comment]
> It would be better to leave both links in the playlist, just changing the name of the old one.

I found this old link. The source is http://www.crtvyradio.xyz/. Obviously this channel is not from Peru, but from Puerto Rico, what do you suggest?

**Comment by Dum4G** [comment]
You can edit https://github.com/iptv-org/iptv/blob/master/channels/pr.m3u in your branch (sguinetti:patch-74) and changes will be present in this PR

**Comment by Dum4G** [comment]
Alternatively I can approve it as it is and the old link will be brought back in your next PR

**Comment by sguinetti** [comment]
Done. I have added Costa Rica (not Puerto Rico as I originally checked).

**Comment by heroku-miraheze** [comment]
"PR" stills Puerto Rico, please, change to Costa Rica, and we are happy to merge it for you.

**Comment by heroku-miraheze** [comment]
"CR" is Costa Rica.

**Review by freearhey — APPROVED** [review]

**Pull request merged**